exonum/src/events/codec.rs

```diff
@@ -149,8 +151,14 @@ mod test {
 
     fn create_encrypted_codecs() -> (MessagesCodec, MessagesCodec) {
         let (public_key, secret_key) = gen_keypair_from_seed(&Seed::new([1; SEED_LENGTH]));
-        let mut params =
-            HandshakeParams::new(public_key, secret_key, SharedConnectList::default(), 1024);
+        let address: SocketAddr = "127.0.0.1:8000".parse().unwrap();
+        let mut params = HandshakeParams::new(
+            public_key,
+            secret_key,
+            SharedConnectList::default(),
+            1024,
+            address,
+        );
```

Comment:
I guess if we have this many parameters then it is time to have a builder for this struct... Though, `HandshakeParams::new()` seems to be used only twice, and in both cases it uses `SharedConnectList::default()` and `1024` as parameters. Maybe we can get rid of these and accept only the keys and the address.